servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/TestBase.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.microsoft.azure.servicebus;
+
+import com.azure.common.test.TestMode;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+
+public class TestBase {
+
+    @BeforeClass
```

Comment:
Thinking a bit about this. For the classes that extend from TestBase, would having this `AfterClass` remove the need for having to perform that `managementClient == null` check?
```java
@AfterClass
public static void skipCleanupIfNotConfigured() {
   Assume.assumeTrue("Nothing to clean up. The test only runs in Live mode.", TestBase.getTestMode() == TestMode.RECORD);
}
```

Reply:
updated